precompiles/avs/abi.json

```diff
@@ -1,35 +1,40 @@
 [
 	{
 		"inputs": [
+			{
+				"internalType": "string[]",
+				"name": "avsOwnerAddress",
+				"type": "string[]"
+			},
 			{
 				"internalType": "string",
 				"name": "avsName",
 				"type": "string"
 			},
 			{
 				"internalType": "string",
-				"name": "avsAddress",
+				"name": "slashContractAddr",
 				"type": "string"
 			},
 			{
-				"internalType": "string",
-				"name": "operatorAddress",
-				"type": "string"
+				"internalType": "string[]",
+				"name": "assetID",
+				"type": "string[]"
 			},
 			{
 				"internalType": "uint64",
 				"name": "action",
 				"type": "uint64"
 			},
 			{
-				"internalType": "string",
-				"name": "avsOwnerAddress",
-				"type": "string"
+				"internalType": "uint64",
+				"name": "minimumDelegation",
+				"type": "uint64"
 			},
 			{
-				"internalType": "string",
-				"name": "assetID",
-				"type": "string"
+				"internalType": "uint64",
+				"name": "unbondingEpochs",
```

Comment:
unbondingEpoch:  it's better named it **unbondingTime** or **unbondingPeriod**

Reply:
fixed